docs/src/test/scala/docs/http/scaladsl/server/HttpsServerExampleSpec.scala

```diff
@@ -49,16 +68,38 @@ abstract class HttpsServerExampleSpec extends AnyWordSpec with Matchers
 
     //#both-https-and-http
     // you can run both HTTP and HTTPS in the same application as follows:
-    val commonRoutes: Route = get { complete("Hello world!") }
+    val commonRoutes: Route = get {
+      complete("Hello world!")
+    }
     Http().newServerAt("127.0.0.1", 443).enableHttps(https).bind(commonRoutes)
     Http().newServerAt("127.0.0.1", 80).bind(commonRoutes)
     //#both-https-and-http
 
     //#bind-low-level-context
-    val routes: Route = get { complete("Hello world!") }
+    val routes: Route = get {
+      complete("Hello world!")
+    }
     Http().newServerAt("127.0.0.1", 8080).enableHttps(https).bind(routes)
     //#bind-low-level-context
 
+    {
+      //#convenience-cert-loading
+      val https: HttpsConnectionContext = ConnectionContext.httpsServer(SSLContextFactory.createSSLContextFromPem(
+        certificatePath = Paths.get("/some/path/server.crt"),
+        privateKeyPath = Paths.get("/some/path/server.key"),
+        caCertificatePaths = Seq(Paths.get("/some/path/serverCA.crt"))
+      ))
+
+      // or from a config block
+      // my-server {
+      //   certificate = "/some/path/server.crt"
+      //   private-key = "/some/path/server.key"
+      //   ca-certificates = ["/some/path/serverCA.crt"]
```

Comment:
Minor note, if I were configuring an HTTPS server, I would call the CA a `clientCA.crt`, since the CA would be used to validate client certificates.

Reply:
Ah, I had the wrong impression here and thought it was to provide the certificate chain, but I do remember now that you need to put the CA-cert together with the cert in the JKS store (and I guess that is why creating the final cert has the roundtrip via JKS, rather than just using the cert directly)

Reply:
Since this may also be used for a client, I wonder if it shouldn't be `trusted-ca-certs` and then be possible to fallback to the default/system store (when empty or through an explicit setting)

Reply:
Yes, agreed.